validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Consensys Software Inc., 2024
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.remote.typedef.handlers;
+
+import static tech.pegasys.teku.infrastructure.json.types.DeserializableTypeDefinition.listOf;
+import static tech.pegasys.teku.validator.remote.apiclient.ValidatorApiMethod.SEND_CONTRIBUTION_AND_PROOF;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import tech.pegasys.teku.spec.datastructures.operations.versions.altair.SignedContributionAndProof;
+import tech.pegasys.teku.validator.remote.typedef.ResponseHandler;
+
+public class SendContributionAndProofsRequest extends AbstractTypeDefRequest {
+  public SendContributionAndProofsRequest(
+      final HttpUrl baseEndpoint, final OkHttpClient okHttpClient) {
+    super(baseEndpoint, okHttpClient);
+  }
+
+  public void submit(final Collection<SignedContributionAndProof> signedContributionAndProofs) {
+    final List<SignedContributionAndProof> requestData =
+        new ArrayList<>(signedContributionAndProofs);
+    postJson(
+        SEND_CONTRIBUTION_AND_PROOF,
+        Collections.emptyMap(),
+        requestData,
+        listOf(requestData.getFirst().getSchema().getJsonTypeDefinition()),
```

Comment:
what if the collection is empty? getFirst will throw

Reply:
<img width="810" alt="image" src="https://github.com/user-attachments/assets/4ddeefad-d5ff-4405-bae9-a151478ae42a">

it is a bit annoying to resolve with the current design.

Reply:
yep this is the same as previous, i had considered that and gone through a couple of iterations including a warning, but decided to leave the behaviour... i can check and see if we can ever call that with 0, but it has never seemed to cause us a problem so i was favouring old behaviour... will dig a tiny bit into old one, not sure we'd ever send signed with 0 (my initial code just returned)

Reply:
hopefully that warning doesnt spam - it shouldnt - it should be a bug if we get that far.